### PR TITLE
[NeuralChat] Support compatible stats format

### PR DESCRIPTION
**Pull request by lvliang-intel**
## Type of Change

bug fix
API not changed

## Description

Support legacy stats format for Cnvg.io

## Expected Behavior & Potential Risk

Users can use legacy stats format.

## How has this PR been tested?

Local test and pre-CI test.

## Dependency Change?

None.

### CODE REVIEW

File: intel_extension_for_transformers/neural_chat/tests/ci/api/test_chatbot_build_api.py [hunk 1]
```diff
@@ -167,6 +167,27 @@ def _run_retrieval(local_dir):
         _run_retrieval(local_dir="/tf_dataset2/inc-ut/instructor-large")
         _run_retrieval(local_dir="/tf_dataset2/inc-ut/bge-base-en-v1.5")
 
+    def test_text_chat_stream_return_stats_with_old_format(self):
```
Comment (hshen14) [inline intel_extension_for_transformers/neural_chat/tests/ci/api/test_chatbot_build_api.py:170]: with_v1_format

File: workflows/chatbot/inference/generate.py [hunk 2]
```diff
@@ -232,7 +238,9 @@ def main():
         use_hpu_graphs=args.use_hpu_graphs,
         use_cache=args.use_kv_cache,
         num_return_sequences=args.num_return_sequences,
-        ipex_int8=args.ipex_int8
+        ipex_int8=args.ipex_int8,
+        return_stats=args.return_stats,
+        legacy_format=args.legacy_format
```
Comment (hshen14) [inline workflows/chatbot/inference/generate.py:243]: format_version

File: workflows/chatbot/inference/generate.py [hunk 3]
```diff
@@ -232,7 +238,9 @@ def main():
         use_hpu_graphs=args.use_hpu_graphs,
         use_cache=args.use_kv_cache,
         num_return_sequences=args.num_return_sequences,
-        ipex_int8=args.ipex_int8
+        ipex_int8=args.ipex_int8,
+        return_stats=args.return_stats,
```
[the remark this answers is not in the document]
Comment (lvliang-intel) [reply workflows/chatbot/inference/generate.py:242]: yes, we need. The default case is no stats info returned.

**Comment by lvliang-intel** [comment]
This UT issue is related with sentence-transformers upgrade.

**Pull request merged**